src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,22 +15,76 @@ redirects:
+
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
+
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
+
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
 The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
 
-The main ways to find and use APM data in infrastructure monitoring are:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring. 
 
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+<Steps>
+    <Step>
+    ## Do you have a problem in your system? 
 
-Infrastructure data appears in <InlinePopover type="apm" /> in the host table on the [APM Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page.
+    You're the engineer responsible for the `Billing Service` application and you get an alert that says, "Error percentage > 45% for at least five minutes on `Billing Service`." The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system. The first indicator of a problem in your system is a high Apdex score, which is a measure user satisfaction. Here you can see that the score is .79 and has triggered a critical violation. Next you're going to check your error rate. Here you can see that the error rate has hit 100%. 
 
-## View logs for your APM and infrastructure data [#logs-context]
+    Based on these two indicators, you know you have a problem. Now you just have to figure out where and why.  
 
-You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
+
+  <img
+    title="Apdex violations"
+    alt="A screenshot depicting critical violations in apdex"
+    src={apmCriticalViolationsinApdex}
+    />
+    </Step>
+    <Step>
+   ## Where does the problem lie? 
```

Comment:
Possible stand in title: Diagnose root cause

(or some other verb besides 'diagnose'. I like 'diagnose' because it implies you're trying to whittle down multiple causes)

Reply:
actually maybe not 'diagnose root cause', but something more specific like 'Determine the source of the error' or some equivalent.

Reply:
done!